samples/cpp/windows/console/samples/intent_recognition_samples.cpp

```diff
@@ -222,3 +222,86 @@ void IntentContinuousRecognitionWithFile()
     recognizer->StopContinuousRecognitionAsync().get();
     // </IntentContinuousRecognitionWithFile>
 }
+
+// Intent recognition using pattern matching and microphone.
+void IntentRecognitionWithPatternMatchingAndMicrophone()
+{
+    // <IntentRecognitionWithPatternMatchingAndMicrophone>
+    // Creates an instance of a speech config with specified subscription key
+    // and service region. Note that in contrast to the other samples this DOES NOT require a
+    // LUIS application.
+    // The default recognition language is "en-us".
+    auto config = SpeechConfig::FromSubscription("YourLanguageUnderstandingSubscriptionKey", "YourLanguageUnderstandingServiceRegion");
+
+    // Creates an intent recognizer using microphone as audio input.
+    auto recognizer = IntentRecognizer::FromConfig(config);
+
+    // Creates a Pattern Matching model and adds specific intents from your model. 
+    // The Id is used to identify this model from others in the collection.
+    auto model = PatternMatchingModel::FromModelId("YourPatternMatchingModelId");
+
+    // Adds some intents to look for specific patterns.
+    model->Intents.push_back({ {"Take me to floor {floorName}", "Go to floor {floorName}", "Take me to {floorName}", "Go to floor {floorName}", "Go to {floorName}", "{floorName}"}, "ChangeFloors"});
```

Comment:
Nitpick: Would it make sense to move the ID for the intent to be the first item? I'm of the opinion that makes the code a little more readable. Probably a breaking change now though